### DISCUSSION TITLE
Numojo short-mid term plans

### DISCUSSION THREAD

**Discussion by lbartworks** [Mojo-Numerics-and-Algorithms-group/NuMojo#112] [Q&A] [answered]
The recent pointer changes in the Mojo definition have quite some impact in Numojo (as well as it had on my code), @MadAlex1997 are you planning to start upgrading the existing code base, wait until Mojo stabilizes more or perhaps something different?

**Comment by MadAlex1997** [comment, marked as answer]
@lbartworks 

We are currently working on a release for Mojo 24.5 that will cover that issue, add a few new features, and clean up and clarify the array generation methods.

In the meantime, you are welcome to use our experimental branch where these changes currently live, although there are still a few rough edges (most notably matmul wasn't working last I checked).

**Comment by shivasankarka** [comment]
@lbartworks Thank you for your interest. We have released NuMojo v0.3 that works with Mojo 24.5 today!
